src/sparkdantic/model.py

```diff
@@ -59,32 +62,32 @@
 
 FieldInfoUnion = Union[FieldInfo, ComputedFieldInfo]
 
-BaseModelOrSparkModel = Union[BaseModel, 'SparkModel']
+BaseModelOrSparkModel = Union[BaseModel, "SparkModel"]
 
 _type_mapping = MappingProxyType(
     {
-        int: 'integer',
-        float: 'double',
-        str: 'string',
-        SecretStr: 'string',
-        bool: 'boolean',
-        bytes: 'binary',
-        SecretBytes: 'binary',
-        list: 'array',
-        dict: 'map',
-        datetime: 'timestamp',
-        date: 'date',
-        Decimal: 'decimal',
-        timedelta: 'interval day to second',
-        UUID: 'string',
-        HttpUrl: 'string',
+        int: "integer",
```

Comment:
We have pre-commit setup on this repo that keeps the code to a certain style. can you run it over your PR so things like quotes don't get changed and become inconsistent with the rest of the code?

Reply:
@mitchelllisle okay, taken care of!